shm/inc/map.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <vector>
+
+#include "island.hpp"
+
+class Map {
+private:
+    std::vector<Island> island_;
```

Comment:
```suggestion
    std::vector<Island> islands_;
```
I think it's more meaningful

Reply:
Correct, thanks